### ISSUE TITLE
Laravel 5.4 implementation?

### ISSUE THREAD

**Issue by metekabak** [Anahkiasen/polyglot#59]
Do you have a plan to upgrade your package for laravel 5.4?

**Comment by Anahkiasen** [comment]
Yes and no, I do plan to update _parts_ of the package to Laravel 5.4 (everything gettext related) but the Eloquent part I think is nowadays handled better by packages like [laravel-translatable](https://github.com/dimsav/laravel-translatable) and will probably be removed from Polyglot unfortunately. I have limited time for OS so it's better for users of the package if I delegate the parts that can be found elsewhere instead of leaving them in without having the time to actually maintain them :/

**Comment by DMeganoski** [comment]
Yes, sorry I had planned on adding 5.4 support shortly after adding 5.3, but considering none of my own applications use 5.4 yet, it hasn't been a high priority. I was more or less waiting for someone to bring the issue up. It doesn't seem there are a whole lot of people using this package.

I was actually unaware of the package Anahkiasen mentioned, I will have to look into it and do some research as to what the similarities and differences are.
